0.13.1 source distribution fails to build, missing header files

Using the newly published 0.13.1 source distribution from PyPi:

```console
$ python setup.py build
running build
running build_py
copying src/uharfbuzz/_version.py -> build/lib.linux-x86_64-3.8/uharfbuzz
running build_ext
building 'uharfbuzz._harfbuzz' extension
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fno-semantic-interposition -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -D_FORTIFY_SOURCE=2 -fPIC -DHB_NO_MT=1 -DHB_EXPERIMENTAL_API=1 -Iharfbuzz/src -I/usr/include/python3.8 -c src/uharfbuzz/_harfbuzz.cpp -o build/temp.linux-x86_64-3.8/src/uharfbuzz/_harfbuzz.o -std=c++11
src/uharfbuzz/_harfbuzz.cpp:654:10: fatal error: hb.h: No such file or directory
  654 | #include "hb.h"
      |          ^~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1
```

The extracted package doesn't seem to have the referenced `hb.h` header file at all. In fact it doesn't include any of the sources needed to build Harfbuzz.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,1 @@
+include harfbuzz/src/*

